src/public/modules/forms/base/componentViews/field-textfield.client.view.html

```diff
@@ -29,6 +29,7 @@
       ng-required="vm.field.required"
       ng-disabled="vm.field.disabled"
       ng-pattern="vm.pattern"
+      ng-style="vm.field.allowPrefill && {'background-color': '#FEF8E3'}"
```

Comment:
would it be possible to avoid the use of `ng-style` and rely on `ng-class` instead, to allow the `background-color` CSS property be specified in a CSS file?

Reply:
edited